lib/rules/prefer-ember-test-helpers.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const { getImportIdentifier } = require('../utils/import');
+
+let hasImportedBlur = undefined;
+let hasImportedFind = undefined;
+let hasImportedFocus = undefined;
```

Comment:
These variable declarations need to be moved inside the `create` function. This is causing test pollution (state leftover between tests) which is why one of your tests is failing.

Reply:
Ahhh you smart :D Fixed!